lightly/utils/benchmarking/metric_callback.py

```diff
@@ -60,7 +60,7 @@ def _append_metrics(
         self, metrics_dict: Dict[str, List[float]], trainer: Trainer
     ) -> None:
         for name, value in trainer.callback_metrics.items():
-            if isinstance(value, float) or (
-                isinstance(value, Tensor) and value.numel() == 1
-            ):
-                metrics_dict.setdefault(name, []).append(float(value))
+            if isinstance(value, Tensor) and value.numel() != 1:
+                # Skip non-scalar tensors.
+                continue
```

Comment:
`trainer.callback_metrics.items()`  returns `(str, Tensor)` tuples and mypy pointed out that `isinstance(value, float)` is impossible if the value is always a tensor. As this check previously passed CI, I believe that either pytorch lightning changed the typing of this function, or I have a different mypy version locally. In any case, I changed the test slightly to make it pass in all cases. The functionality should remain exactly the same :)